tsdb/index/tsi1/index.go

```diff
@@ -89,6 +91,12 @@ var WithMaximumLogFileSize = func(size int64) IndexOption {
 	}
 }
 
+var WithMaximumLogFileAge = func(dur time.Duration) IndexOption {
```

Comment:
Yes good catch! I should test `buildtsi`.

Reply:
We actually don't need it as we use 4 hours as a default - but I realised another good change in buildtsi would be to fully compact the log files so that at least on startup the database is in a good state.